### ISSUE TITLE
yapf breaks Django template blocks

### ISSUE THREAD

**Issue by osya** [google/yapf#529]
`yapf` breaks Django template blocks according to the `column_limit` setting. For example, for `column_limit = 120` source line
`{% extends '_layouts/base.html' %} {% load thumbnail %} {% block title %}All Albums{% endblock title %} {% block headline %}`
converted with `yapf` to the 
```
{% extends '_layouts/base.html' %} {% load thumbnail %} {% block title %}All Albums{% endblock title %} {% block headline
%}
```
It is wrong. `yapf` should never ever breaks Django template blocks

**Comment by gwelymernans** [comment]
I didn't even know this was a thing. :-) What's the format for these templates? Do they appear outside of strings? Are they bracketed by some keywords of some sort?

**Comment by osya** [comment]
Django templates is just a regular .html files with additional elements in curly braces (you can look through how it looks [there](https://docs.djangoproject.com/en/2.0/ref/templates/language/)). So, in .html files strings concludes in {% ... %}, {{ ... }}, {# ... #} should not be braked.

**Comment by gwelymernans** [comment]
I'm a bit confused then. Are you using yapf to reformat your Django files? It's not really built to do that...

**Comment by osya** [comment]
@gwelymernans , agree. I already setup my dev environment to not apply `yapf` to .html files due it is not intended to this. May be it should be written in README more clearly? So this issue may be closed.
